building/trusted_applications.rst

```diff
@@ -227,6 +231,8 @@ Trusted Application properties shall be defined in a header file named
 
 Refer to :ref:`ta_properties` to understand how to configure these macros.
 
+Hint: UUIDs can be generated at https://www.guidgen.com/ or with similar tools.
```

Comment:
With `.. hint::` you'll get a real "Hint notification" on the page (search for *.rst | grep hint).